src/main/java/Duke.java

```diff
@@ -1,10 +1,346 @@
+
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import duke.task.Todo;
+import duke.task.Task;
+import duke.task.TestTask;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.exception.DukeException;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    static int listLength;
+
+    static Task task;
+
+
+    static String userInput;
+    static String command;
+    static String description = "No description is needed for this command";
+    public static final String SEPARATOR = "-----------------------------------";
+    private static ArrayList<Task> tasks;
     public static void main(String[] args) {
```

Comment:
The main method is too long ([>30 LoC](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)).  

Consider ways to simplify the code (e.g. ensuring [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) across the method to abstract long code blocks).